### PR TITLE
Unwanted message removed.

### PR DESCRIPTION
**Pull request by aSquare14**
#807 Resolved

### CODE REVIEW

**Comment by cartothemax** [comment]
Hello @aSquare14 it looks like some unrelated commits got into your branch?  I believe this commit https://github.com/ifmeorg/ifme/pull/808/commits/3e5ac9e1402c4e45215474493aac40ad085cbc51 is the only one that is needed to get rid of that message you are seeing.

File: app/views/groups/_form.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <%= form_for(@group) do |f| %>
   <% if @group.errors.any? %>
     <div class="error_explanation">
-      <h2>t('groups.form.error_save')</h2>
+      <!-- <h2>t('groups.form.error_save')</h2> -->
```
Comment (cartothemax) [inline app/views/groups/_form.html.erb]: I would suggest just deleting this `<h2>` rather then commenting it out.

**Comment by aSquare14** [comment]
Okay. I'll delete it and remove the excess commits.

**Comment by aSquare14** [comment]
I don't know how to remove those commits, do I just make a new PR ?

File: client/app/stories/index.jsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import ChartControl from '../bundles/momentDashboards/components/ChartControl';
 import Logo from '../bundles/shared/components/Logo';
 import Input from '../bundles/shared/components/Input';
 
+<<<<<<< HEAD
```
Comment (Seunadex) [inline client/app/stories/index.jsx]: @aSquare14 You have a couple of merge conflict to fix properly in this file

**Comment by cartothemax** [comment]
No worries @aSquare14! It will be easiest to close this PR and open a new one that has just the removed `<h2>` tag.  

Be sure to update master on your fork of ifme so that you have all of the changes from upstream master.  Let me know if you have any questions! Slack is a good resource to help out as well https://github.com/ifmeorg/ifme/blob/master/CONTRIBUTING.md#contributor-roles

**Comment by aSquare14** [comment]
I was researching online, I read that closing and opening new PRs in orgs is not a good practice .Hence I researched and was able to fix this. Now I just have a single commit and not the unwanted commits. Hope these changes are okay !

**Comment by cartothemax** [comment]
Nice one @aSquare14!

**Comment by julianguyen** [comment]
Can you rebase from master? The Nokogiri issue you're encountering in CircleCI has been recently fixed! Also thank you for working on this 💜 thank you @cartothemax for reviewing :)

**Comment by aSquare14** [comment]
Rebased :D

**Pull request merged**